set.mm

```diff
@@ -465091,7 +465091,8 @@ its graph has a given second element (that is, function value).
 
   That decimal point operator is defined in the next section.
   The bulk of these constructions have originally be proposed by David A.
-  Wheeler on 15-May-2015.
+  Wheeler on 12-May-2015, and discussed with Mario Carneiro in this thread:
+  ~ https://groups.google.com/g/metamath/c/2AW7T3d2YiQ/m/iSN7g87t3ikJ .
```

Comment:
This seems to be the URL to a specific message in the thread. Link to the thread:
https://groups.google.com/g/metamath/c/2AW7T3d2YiQ
(I removed the "/m/...").